routes/home.js

```diff
@@ -15,9 +15,10 @@ async function handleIndexRoute(req, res) {
     title: "Firefox Monitor",
     breach: breach,
   });
-}
+});
 
-router.get("/", handleIndexRoute);
-router.get("/monitor", handleIndexRoute);
+router.get("*", (req, res) => {
+  res.redirect("/");
+});
```

Comment:
A bit curious how this works. It seems like you just may have hijacked the 404 route, so now every broken image and favicon may try loading the homepage and returning a 200 status code. Maybe. 

I really need to try and fix up my dev envy during the all hands so I can try some of this locally again. But I’m almost curious enough to let y’all merge it, and we can test it out on heroku.